ci/matrix.yaml

```diff
@@ -3,6 +3,9 @@ cuda_prev_min: &cuda_prev_min '11.1'
 cuda_prev_max:  &cuda_prev_max  '11.8'
 cuda_curr: &cuda_curr '12.3'
 
+# Minimum for the cuda_next library (yes, var name is confusing.)
+cuda_next_min: &cuda_next_min '12.0'
```

Comment:
This line and those above it show some of the reasons I'm not a fan of the name "cuda next". It sounds like we're talking about CTK. Things like `cuda_prev_min` and `cuda_next_min` being completely unrelated make the infra around this project name confusing and unintuitive.

I'm growing more and more fond of `cccl::preview`. It' not overloaded with anything, and "preview" implies instability, rather than "next" which implies release-candidate.